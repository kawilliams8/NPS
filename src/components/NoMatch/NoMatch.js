import React from 'react';
import { NavLink } from 'react-router-dom';
import Arrowhead from '../../assets/images/US-NationalParkService-Logo.svg';

const NoMatch = () => {
  return (
    <div className="NoMatch"> 
      <h2>Sorry, this page doesn't exist.</h2>
      <h3>Return to the{<NavLink to="/" className="NavText">Welcome Center</NavLink>}</h3>
      <img src={Arrowhead} alt="National Parks Service arrowhead logo" />
    </div>
  )
}

export default NoMatch;