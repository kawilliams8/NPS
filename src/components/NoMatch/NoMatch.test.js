import React from 'react';
import { shallow } from 'enzyme';
import NoMatch from '../NoMatch/NoMatch';

describe('NoMatch', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<NoMatch />)
    expect(wrapper).toMatchSnapshot()
  })
})