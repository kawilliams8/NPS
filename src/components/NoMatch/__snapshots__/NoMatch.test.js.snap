// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch should match snapshot 1`] = `
<div
  className="NoMatch"
>
  <h2>
    Sorry, this page doesn't exist.
  </h2>
  <NavLink
    className="NavText"
    to="/"
  >
    HOME
  </NavLink>
</div>
`;
